src/app/utils/ClientUpdateUtils.ts

```diff
@@ -23,138 +23,146 @@ import { StorageService } from '../services/storage.service';
 import { Injectable } from '@angular/core';
 import { GithubReleaseRequest, GithubReleaseResponse } from '../requests/requests';
 
-export interface ETHClient {
+interface ClientInfo {
     key: string,
     name: string,
-    repo: string
+    repo: string,
+    storageKey: string,
 }
 
-export const ETH2Clients: ETHClient[] = [
-
+export const Clients: ClientInfo[] = [
+    // Execution
     {
         key: "LIGHTHOUSE",
         name: "Lighthouse",
         repo: "sigp/lighthouse",
+        storageKey: "client_updates_lighthouse",
+    },
+
+    {
+        key: "LODESTAR",
+        name: "Lodestar",
+        repo: "chainsafe/lodestar",
+        storageKey: "client_updates_lodestar",
     },
 
     {
         key: "PRYSM",
         name: "Prysm",
         repo: "prysmaticlabs/prysm",
+        storageKey: "client_updates_prysm",
     },
 
     {
         key: "NIMBUS",
         name: "Nimbus",
         repo: "status-im/nimbus-eth2",
+        storageKey: "client_updates_nimbus",
     },
 
     {
         key: "TEKU",
         name: "Teku",
         repo: "ConsenSys/teku",
+        storageKey: "client_updates_teku",
+    },
+    // Consensus
+    {
+        key: "BESU",
+        name: "Besu",
+        repo: "hyperledger/besu",
+        storageKey: "client_updates_besu",
     },
-]
-
 
-export const ETH1Clients: ETHClient[] = [
+    {
+        key: "ERIGON",
+        name: "Erigon",
+        repo: "ledgerwatch/erigon",
+        storageKey: "client_updates_erigon",
+    },
 
     {
         key: "GETH",
         name: "Geth",
         repo: "ethereum/go-ethereum",
-    },
-
-    {
-        key: "OPENETHEREUM",
-        name: "Open Ethereum",
-        repo: "openethereum/openethereum",
+        storageKey: "client_updates_geth",
     },
 
     {
         key: "NETHERMIND",
         name: "Nethermind",
         repo: "NethermindEth/nethermind",
+        storageKey: "client_updates_nethermind",
     },
-
-    {
-        key: "BESU",
-        name: "Besu",
-        repo: "hyperledger/besu",
-    },
-    {
-        key: "ERIGON",
-        name: "Erigon",
-        repo: "ledgerwatch/erigon",
-    },
-]
-
-export const OtherClients: ETHClient[] = [
-
+    // Various
     {
         key: "ROCKETPOOL",
         name: "Rocketpool",
         repo: "rocket-pool/smartnode-install",
+        storageKey: "smart_node_updates",
     },
     {
         key: "MEV-BOOST",
         name: "MEV-Boost",
         repo: "flashbots/mev-boost",
+        storageKey: "mev_boost_updates",
     },
 ]
 
-
-
 const LOCAL_UPDATED_KEY = "mark_clientupdate_completed"
-export const ETH1_CLIENT_SAVED = "setting_client_eth1"
-export const ETH2_CLIENT_SAVED = "setting_client_eth2"
```

Comment:
These two should match the storage keys used in `convertOldToNewClientSettings()`.

Reply:
lgtm